parent/pom.xml

```diff
@@ -134,14 +132,10 @@
         <snappy.version>1.1.7.2</snappy.version>
         <spark.hadoop.version>2.6.5</spark.hadoop.version>
         <spark.version>2.3.0</spark.version>
-        <spring.data.version>1.13.23.RELEASE</spring.data.version> <!-- don't forget to update spring version -->
-        <spring.version>4.3.26.RELEASE</spring.version><!-- don't forget to update spring-data version -->
-        <spring.data-2.0.version>2.0.13.RELEASE</spring.data-2.0.version> <!-- don't forget to update spring-5.0 version -->
-        <spring-5.0.version>5.0.16.RELEASE</spring-5.0.version><!-- don't forget to update spring-data-2.0 version -->
-        <spring.data-2.2.version>2.2.3.RELEASE</spring.data-2.2.version> <!-- don't forget to update spring-5.2 version -->
-        <spring-5.2.version>5.2.3.RELEASE</spring-5.2.version><!-- don't forget to update spring-data-2.2 version -->
+        <spring.version>5.2.21.RELEASE</spring.version><!-- don't forget to update spring-data version -->
+        <spring.data.version>2.2.13.RELEASE</spring.data.version> <!-- don't forget to update spring version -->
         <spring41.osgi.feature.version>4.1.7.RELEASE_1</spring41.osgi.feature.version>
```

Comment:
It seems this property is not used and can be removed.

Reply:
Removed the property